### ISSUE TITLE
CLI Updates

### ISSUE THREAD

**Issue by geofffranks** [starkandwayne/shield#28]
Update the shield CLI to be a little more friendly:

- When listing archives, include the archive state (valid/invalid/expired/etc)
- When listing anything with target data (targets, tasks, jobs, archives), include the agent IP of the target
- When running a restore option on the implicit target, have the client retrieve that info from the DB, and publish it (along with agent IP) vs `restoring to ''`
- When displaying agent data, have the CLI refer to it as the 'Remote Agent' or 'remote_agent', rather than 'SHIELD Agent', or 'agent'

**Comment by qanx** [comment]
* Consistent naming between "Description" and "Summary"

```
$ shield ls schedules
UUID                                   Name            Description   Frequency / Interval (UTC)
====                                   ====            ===========   ==========================
d23e818a-5000-448c-9b03-f179a3592754   Daily Backups                 daily 3am

$ shield show schedule d23e818a-5000-448c-9b03-f179a3592754
UUID:     d23e818a-5000-448c-9b03-f179a3592754
Name:     Daily Backups
Summary:
Timespec: daily 3am
```

**Comment by qanx** [comment]
Resolved with 4e8aff1 & 9cd28ef
